src/modules/blocks/attendees/container.js

```diff
@@ -23,19 +23,13 @@ const mapStateToProps = ( state ) => ( {
 } );
 
 const mapDispatchToProps = ( dispatch ) => ( {
-	...bindActionCreators( actions, dispatch ),
-	onSetDisplayTitleChange: onSetDisplayTitleChange( dispatch ),
-	onSetDisplaySubtitleChange: onSetDisplaySubtitleChange( dispatch ),
+	dispatch,
```

Comment:
let's remove this since we aren't using this in the template.

Reply:
Perfect, done.